tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -63,10 +63,11 @@ def test_deployment(self):
 
         # Download installer
         installer = get_openshift_installer(
-            DEPLOYMENT['installer_version']
+            DEPLOYMENT['installer_version'],
+            force_download=True,
```

Comment:
I think that we should allow to user to avoid to install it all the time by passing some variable from our custom config, so ppl can create their custom config where they will specify to not download each time and take those data from config like:
force_download=DEPLOYMENT['force_download_installer']

I remember that Harish told me that at home he has bad network connection and he want to use the file which was downloaded before so that it's on ppls responsibility to delete the file if the want new one, WDYT?

Reply:
@petr-balogh yeah, this make sense... will update it that way...

Reply:
Fixed in bc1c461.